allensdk/api/queries/mouse_atlas_api.py

```diff
@@ -0,0 +1,142 @@
+# Allen Institute Software License - This software license is the 2-clause BSD
+# license plus a third clause that prohibits redistribution for commercial
+# purposes without further permission.
+#
+# Copyright 2015-2018. Allen Institute. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# 3. Redistributions for commercial purposes are not permitted without the
+# Allen Institute's written permission.
+# For purposes of this license, commercial purposes is the incorporation of the
+# Allen Institute's software into anything for which you will charge fees or
+# other compensation. Contact terms@alleninstitute.org for commercial licensing
+# opportunities.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+
+from allensdk.core import sitk_utilities
+from allensdk.api.cache import Cache, cacheable
+
+from .reference_space_api import ReferenceSpaceApi
+from .grid_data_api import GridDataApi
+from .rma_pager import pageable
+
+
+
+class MouseAtlasApi(ReferenceSpaceApi, GridDataApi):
+
```

Comment:
Maybe put a class-level docstring here with a See also link back to the server-side API docs. 
Similar to: https://github.com/AllenInstitute/AllenSDK/blob/master/allensdk/api/queries/mouse_connectivity_api.py#L48

Reply:
This is where it would end up in the doc build.  Sometimes when you're wading through that stuff (or reading the code directly) it's nice to have a link back to the higher level discussion.
http://alleninstitute.github.io/AllenSDK/allensdk.api.queries.html#module-allensdk.api.queries.mouse_connectivity_api

Along those lines a python notebook example would be a nice-to-have (maybe one exists already).
I think the mouse connectivity one is largely based on the server-side api documentation
http://alleninstitute.github.io/AllenSDK/_static/examples/nb/mouse_connectivity.html